src/schema.ts

```diff
@@ -40,79 +49,10 @@ export type DrumType = "bassdrum" | "hihat" | "snare";
  */
 export type Channel = DrumType | "vibraphone" | "bass";
 
-/**
- * Represents a program on the MMX's programming wheel. Used to store the programmed part of a performance.
- */
-export interface Program {
-	/**
-	 * Information about this program. It can be distinct from a performance's metadata.
-	 */
-	metadata: ProgramMetadata;
-	/**
-	 * This is the "working" state of the machine, outside of a performance.
-	 * When a user is modifying the program, this is how we store what settings they use.
-	 * Whenever the user saves, this field gets updated, and whether they load, this gets read.
-	 * It has almost no relevance when editing a performance since the performance events will
-	 * cause it to be updated dynamically.
-	 *
-	 * ## Example
-	 * ```typescript
-	 * let prog: Program;
-	 * // invert the muted state of the snare
-	 * prog.state.machine.mute.snare = !prog.state.machine.mute.snare;
```

Comment:
Doesn't the machine have a state? Not that a state has a machine? And a mute doesn't have a snare, but a snare might be muted?

Reply:
Confusing name I guess, MachineState is state that is for the whole machine, instead of a particular instrument, based on how the machine is physically laid out.  The mute levers are all controlled separately from the instruments they mute and all use a common mechanism, this the state that represents them mirrors that.